.travis.yml

```diff
@@ -3,20 +3,17 @@ sudo: false
 language: node_js
 matrix:
   include:
-    - node_js: '10'
-      env: INTEGRATION=false
+    - node_js: '11'
```

Comment:
Are you sure you want to remove `10`, as it is a LTS version of node? Maybe add `latest` to the matrix for `11`?

Reply:
@leipert fixed

Reply:
Thank you 🙇